src/main/scala/chiseltest/package.scala

```diff
@@ -233,6 +232,9 @@ package object chiseltest {
     Context().doTimescope(() => contents)
   }
 
+  /** get current elapsedTime in nano second. */
+  def elapsedNanoSeconds: Long = Context().simulatorInterface.elapsedNanoSeconds()
```

Comment:
I don't think this should be part of the general public API.

Reply:
I think maybe it can be an API addition. we can discuss later.

Reply:
API removed